zebrad/tests/acceptance.rs

```diff
@@ -216,13 +229,17 @@ fn start_no_args() -> Result<()> {
     // Make sure the command was killed
     assert!(output.was_killed());
 
+    // Check that we have persistent sled database
+    let cache_dir = tempdir.join("state");
+    assert!(cache_dir.read_dir()?.count() > 0);
```

Comment:
Looks good!

Reply:
looks gorgeous

Reply:
It's also *very* obvious what it's meant to be testing.